liblangutil/Scanner.cpp

```diff
@@ -427,10 +432,11 @@ Token Scanner::scanSlash()
 		{
 			// doxygen style /// comment
 			Token comment;
+			int whitespacesAppended;
 			m_skippedComments[NextNext].location.start = firstSlashPosition;
 			m_skippedComments[NextNext].location.source = m_source;
-			comment = scanSingleLineDocComment();
-			m_skippedComments[NextNext].location.end = sourcePos();
+			tie(comment, whitespacesAppended) = scanSingleLineDocComment();
+			m_skippedComments[NextNext].location.end = sourcePos() - whitespacesAppended;
```

Comment:
This is the actual correction.

Reply:
Wouldn't it be better to pull this line into `scanSingleLineDocComment`?

Reply:
I would say the function should either directly operate on `m_skippedComments[NextNext]` or return a TokenDesc and this part reads

```
m_skippedComments[NextNext] = scanSingleLineDocComment();
```

Reply:
Ok, changed that accordingly.